packages/openapi-generator/src/parser/responses.ts

```diff
@@ -0,0 +1,30 @@
+import { OpenAPIV3 } from 'openapi-types';
+import { $Refs } from '@apidevtools/swagger-parser';
+import { OpenApiSchema } from '../openapi-types';
+import { resolveObject } from './refs';
+import { parseMediaType } from './media-type';
+
+export function parseResponses(
+  responses: OpenAPIV3.ResponsesObject | undefined,
+  refs: $Refs
+): OpenApiSchema {
```

Comment:
👍 nice return type

Reply:
Right? It makes things sooooo much easier! ;)